arangod/RestHandler/RestStatusHandler.cpp

```diff
@@ -60,6 +65,16 @@ RestStatus RestStatusHandler::execute() {
     return RestStatus::DONE;
   }
 
+  bool found;
+  std::string const& overviewStr = _request->value("overview", found);
+  if (found && StringUtils::boolean(overviewStr)) {
+    return executeOverview();
+  } else {
+    return executeStandard(security);
+  }
```

Comment:
all this can be simplified to
``` 
if (_request->parsedValue("overview", false)) {
  return executeOverview();
} else {
  return executeStandard(security);
}
```

Reply:
updated